src/popup/components/Modals/ConfirmRawSign.vue

```diff
@@ -93,6 +104,9 @@ export default defineComponent({
     const activeAccount = getLastActiveProtocolAccount(PROTOCOLS.aeternity);
 
     const dataAsString = computed((): string => popupProps.value?.txBase64?.toString() || '');
+    const isUnknownDapp = computed(() => (
+      !popupProps.value?.app || popupProps.value.app.name === UNKNOWN_SOURCE
```

Comment:
I don't think comparing the app name to `'Unknown source'` string is the best way to establish if the app is "unknown". What if the app will introduce itself as "Unknown source"?
Maybe empty `url` is enough?